### ISSUE TITLE
Captcha-Texte für Registrierung sind weg

### ISSUE THREAD

**Issue by innosmith** [decidim-swiss/decidim.swiss#130]
Entweder durch ein Upgrade, Aktualisierung des lokalen Sprachfiles oder andere Änderung sind bei der Neuregistrierung die drei Captcha-Texte leer, resp. entsprechende Keys und Übersetzungen müssen analog vor der Umstellung wieder hergestellt werden. Danke.

![Image](https://github.com/user-attachments/assets/098fba1a-2052-4333-9945-12f098b7a933)

**Comment by larsUE** [comment]
@innosmith das sind fehlende Keys. Ich kann dir morgen die Keys aus Zürich zukommen lassen.

**Comment by innosmith** [comment]
> @innosmith das sind fehlende Keys. Ich kann dir morgen die Keys aus Zürich zukommen lassen.

Gehören diese nicht in das lokale Sprachfile? Dann macht es am besten gleich Puzzle wegen des Redeployments.

**Comment by nicolefreypuzzle** [comment]
Robin testet, prüft und deployed. Erstmal auf Testumgebung für Zürich, danach Deployment beide Plattformen.

**Comment by innosmith** [comment]
> Robin testet, prüft und deployed. Erstmal auf Testumgebung für Zürich, danach Deployment beide Plattformen.

@nicolefreypuzzle : Ob dies auch die Zürcher Plattformen betrifft, weiss ich nicht. Zumindest auf decidim.swiss liegt das Problem vor.

**Comment by larsUE** [comment]
@innosmith doch, betrifft beide Tenants. Wir hatten das am Standup kurz besprochen und ich habe bereits für beide Tenants einen PR eröffnet. :-)

**Comment by Robin481** [comment]
Dank dem PR hat es das Issue ein bisschen früh geschlossen.
@innosmith OCL auf INT ist der Fix deployed
@larsUE ist auch auf Zürich Prod deployed :)

**Comment by innosmith** [comment]
> Dank dem PR hat es das Issue ein bisschen früh geschlossen. @innosmith OCL auf INT ist der Fix deployed @larsUE ist auch auf Zürich Prod deployed :)

Danke @Robin481, funktioniert auf INT ImpulsKöniz. M.E. wäre jetzt während der Ferienzeit ein guter Zeitpunkt für ein kurzes PROD-Deployment, wenn für alle anderen auch ok. Im August sollten wir dann für ImpulsKöniz möglichst keine Unterbrüche haben.

**Comment by larsUE** [comment]
Passt für mich!

**Comment by nicolefreypuzzle** [comment]
@NicolaNaf Hallo Nicola, aufgrund dieses Issues ist ein PROD-Deployment auf decidim.swiss geplant, was auch euch betrifft. Wäre das ok, wenn wir das z.B. für nächste Woche einplanen würden?
